gsplat/strategy/ops.py

```diff
@@ -46,7 +46,7 @@ def _multinomial_sample(weights: Tensor, n: int, replacement: bool = True) -> Te
 
 @torch.no_grad()
 def _update_param_with_optimizer(
-    param_fn: Callable[[str, Tensor], Tensor],
+    param_fn: Callable[[str, Tensor, bool], Tensor],
```

Comment:
This breaks the API of this `_update_param_with_optimizer` function. Though it is kinda ok because it's an internal function, there seem to be a simple way that wouldn't break it:

Instead of 
```
def param_fn(name: str, p: Tensor, requires_grad: bool) -> Tensor:
        return torch.nn.Parameter(p[sel], requires_grad=requires_grad)
```

We could do
```
def param_fn(name: str, p: Tensor) -> Tensor:
        return torch.nn.Parameter(p[sel], requires_grad=p.requires_grad)
```